docs/Alias/Registration.md

```diff
@@ -6,13 +6,10 @@ An [internal user](../Stakeholders/Internal%20user.md) who does not have an alia
 
 1. An internal user with SSB ID `feedId` and a room server with SSB ID `roomId` are connected to each other via secret-handshake
 1. The internal user chooses a `alias` as a candidate [alias string](Alias%20string.md)
-1. The internal user calls a specific [muxrpc](https://github.com/ssb-js/muxrpc/) `async` API `room.registerAlias(alias, feedId, signature, callback)` where `signature` is a cryptographic signature of the string `=room-alias-registration:${roomId}:${feedId}:${alias}` using `feedId`'s cryptographic keypair, read more about it in the [alias database](Alias%20database.md) spec
+1. The internal user calls a specific [muxrpc](https://github.com/ssb-js/muxrpc/) `async` API `room.registerAlias(alias, signature, callback)` where `signature` is a cryptographic signature of the string `=room-alias-registration:${roomId}:${feedId}:${alias}` using `feedId`'s cryptographic keypair, read more about it in the [alias database](Alias%20database.md) spec
```

Comment:
🐑 ✂️: can you verify the signature? this is probably what you and cryptix were yakking about, but figured i would toss in another pair of scissors

Reply:
Hmm, yes, peers can verify it :eyes: